articles/inter-vlan-routing/index.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/inter-vlan-routing/
+title: Inter-VLAN routing
+description: This article is viewed at helping one to gain the understanding of inter-vlan routing and how to configure it using a packet tracer, add fields `dateCreated` and `lastModified` in the `Todo` data class.
+author: atonya-dennis
+date: 2021-03-16T00:00:00-10:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/inter-vlan-routing/hero.jpg
+    alt: Network Connection example image
+---
+### Concept of Inter-VLAN routing
+As a network administrator one of the tasks is to segment a single switched LAN logically without having to run new cables and making changes to the existing network.The concept of **`VLANS`** and how they are configured to communicate will help one to be able to handle the task at hand with ease.
+<!--more-->
```

Comment:
Add a `Prerequisite` section talking about the target audience for this article, and concepts that the reader must have, to understand this article better